test/regress/tests/router/sql/shard_routing.sql

```diff
@@ -65,7 +65,7 @@ SELECT * FROM xxtt1 a WHERE a.w_id = 21 and j + i != 0;
 SELECT * FROM xxtt1 a WHERE a.w_id = 21 and w_id <= 30 and j + i != 0;
 
 -- check that `INSERT FROM SELECT` works
```

Comment:
I don't understand this change. if `INSERT INTO xx SELECT * FROM xx a WHERE a.w_id = 20;` not works its a regression.